### DISCUSSION TITLE
antv g2 饼图值为0也要显示扇形, 应该如何设置

### DISCUSSION THREAD

**Discussion by ywww** [antvis/G2#5269] [Q&A]
antv g2 饼图值为0也要显示扇形, 应该如何设置

**Comment by pearmini** [comment]
有具体的场景和数据吗？

**Reply by ywww** [reply]
[
    { id: 'c', value: 526 },
    { id: 'sass', value: 0 },
    { id: 'php', value: 0 },
    { id: 'elixir', value: 0 },
    { id: 'rust', value: 54 },
  ]

**Comment by ywww** [comment]
![image](https://github.com/antvis/G2/assets/10680248/8319db7e-39dd-4370-9860-e59dde3979ac)

**Reply by pearmini** [reply]
希望展示成什么样？

**Comment by ywww** [comment]
https://g2.antv.antgroup.com/zh/examples/general/pie/#spider-label
